modules/python/python-types.c

```diff
@@ -124,6 +124,29 @@ py_list_from_list(const gchar *list, gssize list_len)
   return obj;
 }
 
+PyObject
+*py_string_list_from_string_list(const GList *string_list)
+{
+  PyObject *obj = PyList_New(0);
+  if (!obj)
+    {
+      return NULL;
+    }
+
+  for (const GList *elem = string_list; elem; elem = elem->next)
+    {
+      const gchar *string = (const gchar *) elem->data;
+      PyObject *py_string = py_string_from_string(string, -1);
+      if (!py_string || PyList_Append(obj, py_string) != 0)
+        {
+          Py_DECREF(obj);
```

Comment:
if PyList_Append() fails we leak py_string

Reply:
Thanks, fixed!